src/services/user.service.js

```diff
@@ -0,0 +1,52 @@
+// const { v4 } = require('uuid');
```

Comment:
The uuid library is imported but not used. If you don't need it, it's better to remove the import to clean up the code.